java_exception_test.go

```diff
@@ -56,6 +56,16 @@ func TestException(t *testing.T) {
 	doTestException(t, "throw_IndexOutOfBoundsException", "IndexOutOfBoundsException")
 	doTestException(t, "throw_ArrayIndexOutOfBoundsException", "ArrayIndexOutOfBoundsException")
 	doTestException(t, "throw_StringIndexOutOfBoundsException", "StringIndexOutOfBoundsException")
+	doTestException(t, "throw_IllegalFormatWidthException", "1000")
+	doTestException(t, "throw_IllegalFormatConversionException", "7 != test.TestThrowable$TestEnum")
+	doTestException(t, "throw_DuplicateFormatFlagsException", "null;DuplicateFormatFlagsException")
```

Comment:
how about change the message `null;DuplicateFormatFlagsException` to `DuplicateFormatFlagsException` when `detailMessage` is nil?

Reply:
ok I will change it

Reply:
ok ，I will change it